src/powershell/Add-EAReader.psm1

```diff
@@ -0,0 +1,54 @@
+<#
+.SYNOPSIS
+Grants EA Reader permissions to the specified Service Principal or Managed Identity
+
+.PARAMETER PrincipalId
+The Object ID of the Service Principal or Managed Identoty.
+
+.PARAMETER PrincipalTenantId
+The Azure Active Directory Tenant which contains the identity.
+
+.PARAMETER BillingAccountId
+The Billing Account ID (Enrollment ID) to grant permissions against.
+
+.EXAMPLE
+Add-EAReader -principalId 00000000-0000-0000-0000-000000000000 -principalTenantId 00000000-0000-0000-0000-000000000000 -billingAccountId 1234567
+
+Grants EA Reader permissions to the specified Service Principal or Managed Identity
+#>
+function Add-EAReader {
+  param(
+    [Parameter(Mandatory=$true)]
+    [string]$PrincipalId,
+    [Parameter(Mandatory=$true)]
+    [string]$PrincipalTenantId,
+    [Parameter(Mandatory=$true)]
+    [string]$BillingAccountId

```

Comment:
Can we change this to `$Scope` so we can support all billing scopes? We can throw a not supported error for any scopes we don't support yet.